test/polytopes.jl

```diff
@@ -873,4 +873,51 @@
       └─ Point(0.0, 0.0, 1.0)"""
     end
   end
+  @testset "Simplex" begin
+    @testset "construction" begin
```

Comment:
@RomeoV please remove these sub-testsets and try to follow the same format of the other tests for Chain, Polygon, Polyhedra